util/libcrypto.num

```diff
@@ -5469,6 +5469,9 @@ OSSL_sleep                              ?	3_2_0	EXIST::FUNCTION:
 OSSL_get_thread_support_flags           ?	3_2_0	EXIST::FUNCTION:
 OSSL_set_max_threads                    ?	3_2_0	EXIST::FUNCTION:
 OSSL_get_max_threads                    ?	3_2_0	EXIST::FUNCTION:
+EVP_RAND_CTX_up_ref                     ?	3_0_7	EXIST::FUNCTION:
+RAND_set0_public                        ?	3_0_7	EXIST::FUNCTION:
+RAND_set0_private                       ?	3_0_7	EXIST::FUNCTION:
```

Comment:
Wrong version numbers - The correct is 3_1_0

Reply:
fixed and relocated to a better place.